core/tests/protractor_utils/AdminPage.js

```diff
@@ -93,10 +93,10 @@ var AdminPage = function() {
     };
 
     this.reloadCollection = async function(collectionId) {
-      this.get();
-      await (
-        await reloadCollectionButtons.get(collectionId)
-      ).click();
+      await this.get();
+      var reloadCollectionButton = await (
```

Comment:
await is not required here because get() doesn't return a promise (see [this](https://www.protractortest.org/#/api?view=ElementArrayFinder.prototype.get)).